extensions-core/datasketches/src/test/java/org/apache/druid/query/aggregation/datasketches/quantiles/sql/DoublesSketchSqlAggregatorTest.java

```diff
@@ -692,13 +692,13 @@ public void testDoublesSketchPostAggsPostSort()
                               0.5
                           ),
                           new DoublesSketchToQuantilePostAggregator(
-                              "s1",
-                              new FieldAccessPostAggregator("s0", "a0:agg"),
+                              "_p1",
+                              new FieldAccessPostAggregator("_p0", "a0:agg"),
```

Comment:
Why did these change?

Reply:
there are 2 callsites to create the postaggregates - one of them uses `p` and the other `s` as prefix; I was getting almost 100% matching plans with column alias mistmatches because of the different prefixes - so I've changed that; as the main importance of them is to have them unique.

I've today tried to remove the change of that prefix thing - and I think that change might not be needed in this patch.

I've update the PR